modules/administration-guide/pages/repairing-the-devworkspace-operator.adoc

```diff
@@ -0,0 +1,88 @@
+:_content-type: PROCEDURE
+:navtitle: Repairing the {devworkspace} Operator
+:description: Repairing the {devworkspace} Operator
+:keywords: devworkspace, operator, repair
+:page-aliases:
+
+[id="repairing-the-devworkspace-operator_{context}"]
+= Repairing the {devworkspace} Operator
+
+.Prerequisites
+
+* An active `{orch-cli}` session as a cluster administrator to the destination {orch-name} cluster. See {orch-cli-link}.
+
+.Procedure
+
+. Delete the `devworkspace-controller` namespace that contains the failing pod.
+
+. Update `DevWorkspace` and `DevWorkspaceTemplate` CRDs by setting the conversion strategy to `None`:
++
+image::repairing-dwo/looking-up-custom-resource-definitions.png[]
```

Comment:
This is a file name, so it doesn't matter much, but still: don't use phrasal verbs in docs.